### ISSUE TITLE
`/signalk` returning weird version output

### ISSUE THREAD

**Issue by ph1l** [SignalK/signalk-server-java#40]
Looks like there is a bug in the endpoints data. (http://signalk.org/specification/master/urls_etc.html#versioning)

```
elektron@x61s:~/code/python-signalk-client$ curl -s http://localhost:8080/signalk | jq .
{
  "endpoints": {
    "1.": {
      "signalk-tcp": "tcp://localhost:55555",
      "signalk-udp": "udp://localhost:55554",
      "nmea-tcp": "tcp://localhost:55557",
      "nmea-udp": "udp://localhost:55556",
      "signalk-http": "http://localhost:8080/signalk/v1/api/",
      "mqtt": "mqtt://localhost:1883",
      "signalk-ws": "ws://localhost:3000/signalk/v1/stream",
      "stomp": "stomp+nio://localhost:61613",
      "version": ".0.0"
    }
  }
}
```

* `endpoints.v1` should exist, but doesn't.
* `endpoints.v1.version` of `.0.0` doesn't make sense

**Comment by rob42** [comment]
Its an artifact of the old version number, which included the 'v', eg v1.0.0.
The code truncates the the wrong bits now. Fixing

**Comment by rob42** [comment]
Fixed in JIT-Deploy-34
